include/drivers/cadence/cdns_uart.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2014, Xilinx Inc.
+ * Written by Edgar E. Iglesias.
```

Comment:
On Wed, 2016-03-30 at 08:23:27 -0700, danh-arm wrote:
> > @@ -0,0 +1,74 @@
> > +/*
> > + * Copyright (c) 2014, Xilinx Inc.
> > + * Written by Edgar E. Iglesias.
> 
> Please use the standard boilerplate text

Yep, this one slipped through. I'll fix this.

	Sören